webapp/src/components/conference/index.ts

```diff
@@ -1,17 +1,16 @@
 import {connect} from 'react-redux';
 import {bindActionCreators, Dispatch} from 'redux';
 
-import {GlobalState} from 'mattermost-redux/types/store';
 import {GenericAction} from 'mattermost-redux/types/actions';
 
-import {id as pluginId} from '../../manifest';
+import {GlobalState} from '../../types';
 import Conference from './conference';
 import {openJitsiMeeting} from '../../actions';
 
 function mapStateToProps(state: GlobalState) {
     return {
-        post: (state as any)[`plugins-${pluginId}`].openMeeting,
-        jwt: (state as any)[`plugins-${pluginId}`].openMeetingJwt
+        post: state['plugins-jitsi'].openMeeting,
```

Comment:
Any reason for this and post_type_jitsi/index.ts to use directly `jitsi` instead of `pluginId`?